python/CMakeLists.txt

```diff
@@ -330,12 +336,38 @@ endif()
 
 set(CYTHON_EXTENSIONS
   lib
-)
+  )
 
 set(LINK_LIBS
   arrow_shared
   arrow_python_shared
-)
+  )
+
+if (PYARROW_BUILD_CUDA)
+  ## Arrow CUDA
+  find_package(ArrowCuda)
+  if(NOT ARROW_CUDA_FOUND)
+    message(FATAL_ERROR "Unable to locate Arrow CUDA libraries")
+  else()
+    if (PYARROW_BUNDLE_ARROW_CPP)
+      bundle_arrow_lib(ARROW_CUDA_SHARED_LIB
+	ABI_VERSION ${ARROW_ABI_VERSION}
+	SO_VERSION ${ARROW_SO_VERSION})
+      if (MSVC)
+	bundle_arrow_implib(ARROW_CUDA_SHARED_IMP_LIB)
```

Comment:
Shouldn't this be indented?

Reply:
It is, but with a tab that shows differently in my emacs. Replaced tab with spaces.